packages/api-specification/interface/screen.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace ssf {
+  export abstract class Screen {
+    static getDisplays(): Promise<Array<Display>>;
+  }
+
+  /**
+   * Information about the users display. Note that the Browser API does not support multiple
+   * displays, so it assumes the display the browser is running in is the only display.
+   */
+  export interface Display {
+    /**
+     * Unique Id of the display.
+     */
+    id: string;
```

Comment:
Just out of interest, where might I use this ID? might be worth mentioning in the comment for this property.